datastore-design/go-grpc-demo/examples/clients/python/Dockerfile.client

```diff
@@ -4,29 +4,33 @@ SHELL ["/bin/bash", "-eux", "-o", "pipefail", "-c"]
 
 ENV CLIENT_PATH=/clients/python
 
-COPY examples/clients/python/requirements.txt "${CLIENT_PATH}/requirements.txt"
-
 # hadolint ignore=DL3013
 RUN apt-get update \
     && apt-get -y upgrade \
-    && pip install --no-cache-dir --upgrade pip \
-    && pip install --no-cache-dir --upgrade -r "${CLIENT_PATH}/requirements.txt" \
+    && apt-get -y install build-essential \
     # Cleanup
     && rm -rf /usr/tmp  \
     && apt-get autoremove -y \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
 COPY protobuf/datastore.proto /protobuf/datastore.proto
+COPY examples/clients/python/requirements.txt "${CLIENT_PATH}/requirements.txt"
+
+RUN pip install --no-cache-dir --upgrade pip \
+    && pip install --no-cache-dir --upgrade -r "${CLIENT_PATH}/requirements.txt"
 
 # Compiling the protobuf file
 RUN python -m grpc_tools.protoc  \
     --proto_path=protobuf protobuf/datastore.proto  \
     --python_out=$CLIENT_PATH  \
     --grpc_python_out=$CLIENT_PATH
 
+COPY examples/test-data/KNMI/20221231.nc "/test-data/KNMI/20221231.nc"
+COPY examples/clients/python/parameters.py "${CLIENT_PATH}/parameters.py"
 COPY examples/clients/python/client.py "${CLIENT_PATH}/client.py"
+COPY examples/clients/python/client_knmi_station.py "${CLIENT_PATH}/client_knmi_station.py"
 
 WORKDIR ${CLIENT_PATH}
 SHELL ["/bin/bash", "-eu", "-o", "pipefail", "-c"]
-CMD ["python", "client.py"]
+CMD ["/bin/bash", "-c", "./client_knmi_station.py; ./client.py"]
```

Comment:
Should this instead be `"./client_knmi_station.py && ./client.py"` so that the second command doesn't run if the first one fails?

Reply:
I haven't made client.py use the data inserted by `client_knmi_station.py`. `client_knmi_station.py` does not touch the `callGetObservations`. Initially, I thought to run them both, but I can change it to `&&`, that is fine for me too.